node.gyp

```diff
@@ -800,6 +804,8 @@
         'common.gypi',
       ],
 
+      'libraries': ['../../deps/milo/libmilo.a'],
```

Comment:
There is something wrong with the build step. You have committed a `.a` binary file and you are referring to it here. Binary files should not be committed to the repo as they are not portable across our target environments.

Reply:
Ignore that for now. That part is completely to be reworked and so far I only tried to make it build on my arch (MacOS M2).

Anyway, Milo is written in Rust.
In theory, given we install the Rust toolchain in our target environment, we could build and include it locally.
But at the end of the day it will result in a `.a` file that will be bundled inside Node.

My original intention instead is to build and release Milo on GH for each platform Node support. Then configure the Node build system to include the right file.

Needless to say, Milo is also an attempt to see if this approach works, which would open the room for similar inclusions in Node.